gatsby_fp/src/components/fieldComponents/profSocieties.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+
+export default function ProfSocieties ({profSocietiesList}){
+  return(
+    <>
+    <div className="sectiontitle">Professional Societies</div>
+        <div>
+          <ul>
+          {profSocietiesList.map((element, index) => (
+            // <li key={index} dangerouslySetInnerHTML ={{__html: 
+            //   `${element.orgabbr} - ${element.org} (${element.dtm_start} ${element.dtd_start} 
+            //   ${element.dty_start} - ${element.dtm_end} ${element.dtd_end} ${element.dty_end})
+            //   <br>${element.status}`}}>
+            // </li>
+            <li key={index} dangerouslySetInnerHTML ={{__html:` 
```

Comment:
We should avoid using indices as keys. (https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)